libs/SmartSync/src/com/salesforce/androidsdk/smartsync/manager/SyncManager.java

```diff
@@ -629,7 +629,14 @@ public RestResponse sendSyncWithSmartSyncUserAgent(RestRequest restRequest) thro
         return restClient.sendSync(restRequest, new HttpAccess.UserAgentInterceptor(SalesforceSDKManager.getInstance().getUserAgent(SMART_SYNC)));
     }
 
-	/**
+    /**
```

Comment:
The new sync down target needs to read from smartstore to decide what to download from the server.